lms/djangoapps/discussion/models.py

```diff
@@ -0,0 +1,16 @@
+
+from django.db import models
+from opaque_keys.edx.django.models import LearningContextKeyField, UsageKeyField
+
+
+class ExternalDiscussionsIdMapping(models.Model):
+    """
+    Store a mapping from internal discussions context to external discussions context
+    """
+    context_key = LearningContextKeyField(max_length=255, db_index=True, null=False)
+    usage_key = UsageKeyField(max_length=255, db_index=True, null=False)
+    external_discussion_id = models.CharField(max_length=255, db_index=True, null=False)
+
+    class Meta(object):
+        unique_together = ("context_key", "usage_key")
+        index_together = ("context_key", "usage_key")
```

Comment:
There is a new way to define these via "indexes" and "constraints". Since the old approach might get deprecated in newer versions of Django, please switch this to using the latest syntax.

Reply:
This is done